_posts/0000-01-02-prince1998.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
*You got a dream. You gotta protect it. People can't do something themselves, they wanna tell you you can't do it. If you want something, go get it.*